index.json

```diff
@@ -1 +1 @@
: "from sourced.ml.models import BOW\nbow = BOW().load(%s)\nprint(\"Number of documents:\", len(bow))\nprint(\"Number of tokens:\", len(bow.tokens))", "license": ["", "none"]}, "1e3da42a-28b6-4b33-94a2-a5671f4102f4": {"created_at": "2017-06-19 09:16:08.942880", "parent": "", "dependencies": ["f64bacd4-67fb-4c64-8382-399a8e7db52a"], "references": [["Similarity of GitHub Repositories by Source Code Identifiers", "http://vmarkovtsev.github.io/techtalks-2017-moscow/#"]], "extra": {"Number of (sub)tokens": "999,424", "Number of repositories": "112,273", "Data collection date": "October 2016"}, "description": "Bags of identifiers generated from 140,000 most starred projects on GitHub in October 2016 - ~112k after deduplication.", "url": "https://storage.googleapis.com/models.cdn.sourced.tech/models%2Fbow%2F1e3da42a-28b6-4b33-94a2-a5671f4102f4.asdf", "license": ["", "undecided"], "size": "380.8 MB", "code": "from sourced.ml.models import BOW\nbow = BOW().load(%s)\nprint(\"Number of documents:\", len(bow))\nprint(\"Number of tokens:\", len(bow.tokens))", "version": [1, 0, 0]}}, "id_splitter_nn": {"522bdd11-d1fa-49dd-9e51-87c529283418": {"created_at": "2019-04-12 10:59:58", "datasets": [[]], "dependencies": [], "description": "Model that contains source code identifier splitter BiLSTM weights.", "environment": {"packages": [["Jinja2", "2.10.1"], ["Keras", "2.2.4"], ["Keras-Applications", "1.0.7"], ["Keras-Preprocessing", "1.0.9"], ["MarkupSafe", "1.1.1"], ["PyStemmer", "1.3.0"], ["PyYAML", "5.1"], ["absl-py", "0.7.1"], ["asdf", "2.3.3"], ["astor", "0.7.1"], ["bblfsh", "2.12.7"], ["certifi", "2019.3.9"], ["chardet", "3.0.4"], ["clint", "0.5.1"], ["dulwich", "0.19.11"], ["gast", "0.2.2"], ["grpcio", "1.19.0"], ["grpcio-tools", "1.19.0"], ["h5py", "2.9.0"], ["humanize", "0.5.1"], ["idna", "2.8"], ["jsonschema", "2.6.0"], ["modelforge", "0.11.1"], ["numpy", "1.16.2"], ["pandas", "0.24.2"], ["pip", "19.0.3"], ["protobuf", "3.7.1"], ["pygtrie", "2.3"], ["python-dateutil", "2.8.0"], ["pytz", "2019.1"], ["requests", "2.21.0"], ["scipy", "1.2.1"], ["semantic-version", "2.6.0"], ["setuptools", "41.0.0"], ["six", "1.12.0"], ["sourced-ml", "0.8.3"], ["spdx", "2.5.0"], ["tensorflow-estimator", "1.13.0"], ["tensorflow-gpu", "1.13.1"], ["termcolor", "1.1.0"], ["urllib3", "1.24.1"]], "platform": "Linux-4.15.0-47-generic-x86_64-with-debian-stretch-sid", "python": "3.6.0 (default, Jan  7 2019, 11:18:05) [GCC 5.4.0 20160609]"}, "license": "ODbL-1.0", "metrics": {}, "references": [[]], "series": "rnn-splitter", "tags": ["identifiers_splitter"], "vendor": "source{d}", "version": [1, 0, 0], "code": "from sourced.ml.models.id_splitter import IdentifierSplitterNN\nimport modelforge.backends\nid_splitter = IdentifierSplitterNN().load(%s, backend=modelforge.backends.create_backend())\nid_splitter.split(identifiers)", "extra": {}, "size": "8.7 MB", "url": "https://storage.googleapis.com/models.cdn.sourced.tech/models%2Fid_splitter_nn%2F522bdd11-d1fa-49dd-9e51-87c529283418.asdf"}}}}
```

Comment:
I know that what I will ask is not really related, but we really need to pretty-format JSON here.

This means editing https://github.com/src-d/modelforge/blob/master/modelforge/index.py#L202 (**and maybe other places**) to supply `sort_keys=True, indent=4`

Reply:
https://github.com/src-d/modelforge/pull/91

Reply:
@vmarkovtsev